stdlib.md

```diff
@@ -39,15 +39,15 @@ Output:
 
 <pre><code>{% include "./programs/stdlib_logging.txt" %}</code></pre>
 
-If you do not have the `cat` command, then you can just open the `test.log` file in a text editor.
+I've used the `cat` command in the command line to read the 'test.log' file, but if you don't have the `cat` command, you can just open the `test.log` file in a text editor.
```

Comment:
Please avoid the use of "I", the text has always been written in the context of the reader, *not* the context of the author.

Reply:
That makes sense.  How about:
"The `cat` command was used in the command line to read the 'test.log' file.  If the `cat` command is not available, you can just open the `test.log` file in a text editor instead."

Reply:
s/was/is - prefer present tense.

Reply:
s/is not available to you/is not available/